packages/sdk-core/src/model/report/BacktraceAttachment.ts

```diff
@@ -0,0 +1,13 @@
+export type BacktraceInMemoryAttachment = {
```

Comment:
why are we not using `interface` here in these options?

Reply:
Pushed adjusted code base on our conversation